node-commons/src/main/java/org/apache/iotdb/commons/partition/SchemaPartition.java

```diff
@@ -163,11 +173,61 @@ public void createSchemaPartition(
         .put(seriesPartitionSlot, regionReplicaSet);
   }
 
-  public void serialize(ByteBuffer buffer) {
-    // TODO: Serialize SchemaPartition
+  public void serialize(ByteBuffer buffer) throws IOException, TException {
```

Comment:
Don't forget to add tests for serialize~

Reply:
fixed